src/site/content/en/blog/optimizing-content-efficiency-loading-third-party-javascript/index.md

```diff
@@ -434,12 +434,9 @@ out of the [critical path](/critical-rendering-path/). Note that this
 approach will still block the `onload` event so try not to attach critical
 functionality to `onload`.
 
-{% Aside %}
-Chrome is also exploring support for [Feature Policy](https://www.chromestatus.com/feature/5694225681219584) - a set of
-policies allowing a developer to selectively disable access to certain browser
-features. This can prevent third-party content introducing unwanted behaviors to
-a site.
-{% endAside %}
+Chrome also supports [Permissions Policy](https://www.chromestatus.com/feature/5694225681219584)
```

Comment:
let's link to https://developer.mozilla.org/en-US/docs/Web/HTTP/Permissions_Policy instead here